_posts/2020-01-06-programming-tutorials.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Why Programmers Should Write Technical Tutorials
+date: 2020-01-06 22:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Programming]
+image: /assets/article_images/roman-mager-59976.jpeg
+image2: /assets/article_images/roman-mager-59976.jpeg
+author: Valentino Urbano
+---
+
+Everyone has read a tutorial on a programming blog somewhere on some topic they were interested in, but few programmers have a blog and even fewer have ever written a tutorial on it. I've written before at length about why you should have a personal blog so I won't tackle that topic here, but only focus on why you should start thinking about writing tutorials.
```

Comment:
This whole paragraph gave me a headache... Sentences here are too long and confusing.

Reply:
Better or still off?